CONTRIBUTING.md

```diff
@@ -64,3 +64,55 @@ Other Tips
     your existing commit message by running:
 
         $ git commit --amend
+
+Checklist for Contributed Code
+------------------------------
+
+A pull request for a new feature will be reviewed to see if it meets the following requirements.  For any pull request, an astropy maintainer can help to make sure that the pull request meets the requirements for inclusion in the package.  
+
+**Scientific Quality**
```

Comment:
Okay.  Maybe just a note somewhere about "where applicable"?  Basically, these recommendations *don't* say how to evaluate something if it is not relevant to Astronomy.  Maybe just common sense applies here though.